### DISCUSSION TITLE
Collaboration with the WoT IG and WG

### DISCUSSION THREAD

**Discussion by egekorkan** [w3c-cg/wot-cg#6] [General]
Here we want to start a discussion that should collect general ideas on how we can collaborate with the WoT IG and WG. We want to have a single public image for WoT and not give the impression that the CG is doing things that can conflict with the IG and WG.

Feel free to provide your opinions here

**Comment by relu91** [comment]
As explained in https://lists.w3.org/Archives/Public/public-web-of-things/2022Oct/0002.html this discussion is locked until further notice.

**Comment by relu91** [comment]
Lately, we started the conversation again, here is a relevant PR: https://github.com/w3c/wot-cg/pull/93
